### DISCUSSION TITLE
Add a callback for loading

### DISCUSSION THREAD

**Discussion by velikodniy** [posativ/isso#778] [Q&A]
Is there any way to set a callback for the moment when `fetchComments` finish loading comments? It's important for parsing mathematical formulae in comments with [MathJax](https://www.mathjax.org/).

Or is there a way to load `embed.min.js` without implicit call of `fetchComments`?

**Comment by angelacastaneda** [comment]
Was there ever any movement on this? did you ever figure out how to add mathjax (or KaTeX) support to isso?
